src/components/ButtonPanel.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Button from './Button'
+
+const ButtonPanel = () => (
+  <div className="button-container">
+    <div className="button-container-div">
+      <Button name="AC" />
+      <Button name="+/-" />
+      <Button name="%" />
+      <Button name="÷" />
+    </div>
+    <div className="button-container-div">
+      <Button name="7" />
+      <Button name="8" />
+      <Button name="9" />
+      <Button name="X" />
+    </div>
+    <div className="button-container-div">
+      <Button name="4" />
+      <Button name="5" />
+      <Button name="6" />
+      <Button name="-" />
+    </div>
+    <div className="button-container-div">
+      <Button name="1" />
+      <Button name="2" />
+      <Button name="3" />
+      <Button name="+" />
+    </div>
+    <div className="button-container-div">
+      <Button name="0" />
+      <Button name="." />
+      <Button name="=" />
+    </div>
```

Comment:
- [ ] Please create an array for button groupings and map through it displaying the buttons. It's neater and concise that way.